### PR TITLE
make lint changes fixed

### PR DESCRIPTION
**Pull request by yashrsharma44**
Fixed the make lint issues created.

Let me know if any more changes are required.
Signed-off-by: Yash Sharma <yashrsharma44@gmail.com>

### CODE REVIEW

File: providers/openmetrics/server_metrics.go [hunk 1]
```diff
@@ -133,13 +133,13 @@ func (m *ServerMetrics) preRegisterMethod(serviceName string, mInfo *grpc.Method
 	methodName := mInfo.Name
 	methodType := string(typeFromMethodInfo(mInfo))
 	// These are just references (no increments), as just referencing will create the labels but not set values.
-	m.serverStartedCounter.GetMetricWithLabelValues(methodType, serviceName, methodName)
-	m.serverStreamMsgReceived.GetMetricWithLabelValues(methodType, serviceName, methodName)
-	m.serverStreamMsgSent.GetMetricWithLabelValues(methodType, serviceName, methodName)
+	_, _ = m.serverStartedCounter.GetMetricWithLabelValues(methodType, serviceName, methodName)
```
Comment (yashrsharma44) [inline providers/openmetrics/server_metrics.go:136]: wondering if this makes sense 🤔

**Comment by bwplotka** [comment]
Can you merge to `v2`? Also rebase with latest changes?

**Comment by bwplotka** [comment]
Oh actually it's to my PR. I need to rebase then first! (:

**Comment by yashrsharma44** [comment]
#394 has refactored the shortcomings addressed by this PR. Closing this.

**Pull request closed without merging**